### ISSUE TITLE
While importing cdms2 error pops up: ImportError: cannot import name 'rank'

### ISSUE THREAD

**Issue by JiveshDixit** [CDAT/cdms#396]
I am using WSL Ubuntu on windows 10.
While importing a CDAT library cdms2, MV2 and vcs i am facing below mentioned error.

` python3 -c "import cdms2"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/jiveslinux/miniconda3/envs/cdat81/lib/python3.6/site-packages/cdms2/__init__.py", line 32, in <module>
    from .dataset import createDataset, openDataset, useNetcdf3  # noqa
  File "/home/jiveslinux/miniconda3/envs/cdat81/lib/python3.6/site-packages/cdms2/dataset.py", line 20, in <module>
    from .coord import FileAxis2D, DatasetAxis2D
  File "/home/jiveslinux/miniconda3/envs/cdat81/lib/python3.6/site-packages/cdms2/coord.py", line 17, in <module>
    from .fvariable import FileVariable
  File "/home/jiveslinux/miniconda3/envs/cdat81/lib/python3.6/site-packages/cdms2/fvariable.py", line 7, in <module>
    from .variable import DatasetVariable
  File "/home/jiveslinux/miniconda3/envs/cdat81/lib/python3.6/site-packages/cdms2/variable.py", line 17, in <module>
    from .avariable import AbstractVariable
  File "/home/jiveslinux/miniconda3/envs/cdat81/lib/python3.6/site-packages/cdms2/avariable.py", line 1970, in <module>
    from . import MV2 as MV  # noqa
  File "/home/jiveslinux/miniconda3/envs/cdat81/lib/python3.6/site-packages/cdms2/MV2.py", line 12, in <module>
    from numpy.ma import indices, innerproduct, masked, put, putmask, rank, ravel  # noqa
ImportError: cannot import name 'rank'`

**Comment by muryanto1** [comment]
Hello, please pick up our unofficial release of cdat 8.2. This includes cdms2, mv2, vcs.
```
wget https://repo.anaconda.com/miniconda/Miniconda3-4.7.12.1-Linux-x86_64.sh -O Miniconda3-4.7.12.1-Linux-x86_64.sh
bash ./Miniconda3-4.7.12.1-Linux-x86_64.sh -b -p miniconda3
source miniconda3/etc/profile.d/conda.sh
conda activate base

conda create -n cdat82 -c cdat/label/v82 -c conda-forge "cdat=8.2" "python=3.7" "numpy=1.17" mesalib="18.3.1" jupyter notebook ipywidgets
conda activate cdat82
```

**Comment by jasonb5** [comment]
Closing will be included in CDMS2 3.1.5 and CDAT 8.2.1/
